index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="titre">Apple Pie</div>
+    </header>
+
+    <section class="main">
+      <div>
+        This was my grandmother's apple pie recipe. I have never seen another
+        one quite like it. It will always be my favorite and has won me several
+        first place prizes in local competitions. I hope it becomes one of your
+        favorites as well!
+        <br /><br />
```

Comment:
adding <br> tags is not good practice, it's better to add padding or margin to position items